pyincore/analyses/housingrecovery/housingrecovery.py

```diff
@@ -176,17 +176,8 @@ def run(self):
         hse_rec_fin2 = pd.DataFrame(hse_rec_fin_index, columns=index_yr)
         hse_recov = pd.concat([hse_recov, hse_rec_fin1, hse_rec_fin2], axis=1)
 
-<<<<<<< HEAD
         column_to_save = ["guid", "d_vacationct", "mhhinck", "pminoritybg", "dmg", "value_loss"] + bval_yr + index_yr
         hse_recov[column_to_save].to_csv(result_name + ".csv", index=False, float_format="%.2f")
-=======
-        if "dmg" in hse_recov.columns:
```

Comment:
So this whole part doesn't need to be in the code, right?

Reply:
Correct, based on Michal's #99 PR

Reply:
I don't see anywhere where we set the output, shouldn't there be a self.set_result_csv_data(...)? This is missing from the original branch #99 as well.